modules/webclient/src/main/scala/scaladex/client/Client.scala

```diff
@@ -84,6 +96,31 @@ object Client {
               }
           }
       }
+
+      val repoRequest: Request = new Request(
+        s"https://api.github.com/repos/$organization/$repository",
+        new RequestInit {
+          headers = headersWithCreds.toJSDictionary
+        }
+      )
+      fetch(repoRequest).toFuture
+        .flatMap { res =>
+          if (res.status == 200) {
+            res.text().toFuture.map(extractDefaultBranch)
+          } else { Future.successful("master") }
+        }
+        .map { res =>
+          val resJson = js.JSON.parse(res)
+          val branch = resJson.asInstanceOf[Repo].default_branch
+          fixImages(branch, organization, repository)
+        }
```

Comment:
I deployed it to https://index-dev.scala-lang.org/scalacenter/bloop but it does not work.

It seems there is a problem here because it tries again to parse the default_branch to JSON.

```suggestion
        .map(branch => fixImages(branch, organization, repository))
```

Reply:
Hmmm weird. I tried it locally.

Reply:
Ah no. My bad. Let me see.